OWNERS_ALIASES

```diff
@@ -148,6 +148,15 @@ aliases:
     - jeremyrickard
     - liggitt
     - micahhausler
+  wg-node-lifecycle-leads:
```

Comment:
@BenTheElder I think this might be the only place where it is useful to add the WG alias.